js/views/playingScreen.js

```diff
@@ -24,6 +24,7 @@ var PlayingScreen = React.createClass({
   getInitialState: function() {
     this.isMobile = this.props.controller.state.isMobile;
     this.browserSupportsTouch = this.props.controller.state.browserSupportsTouch;
+    this.selectableScreenClassName = 'oo-state-screen-selectable';
```

Comment:
As I mentioned before, I think it would be preferable to set the event handlers on the element that requires them instead of the document, but if the events are needed at the document level then we should probably store this class in a constant since it's used in several places.

Reply:
For vr 360 we need to listen this event on document.